testsuite/features/step_definitions/command_steps.rb

```diff
@@ -449,8 +449,7 @@
 end
 
 When(/^I execute mgr\-bootstrap "([^"]*)"$/) do |arg1|
-  arch = 'x86_64'
-  $command_output, _code = $server.run("mgr-bootstrap --activation-keys=1-SUSE-KEY-#{arch} #{arg1}")
+  $command_output, _code = $server.run("mgr-bootstrap #{arg1}")
```

Comment:
So we don't need to provide the activation key anymore here?

Reply:
Yes, we do not provide a default key there, anymore. Instead, we can now specify which key we want:

```gherkin
When I execute mgr-bootstrap "--activation-keys=1-PROXY-KEY-x86_64 --script=bootstrap-proxy.sh"
```